helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -62,6 +64,19 @@ internal object PathExcludeGenerator {
     }
 }
 
+private val REGEX_SPECIAL_CHARACTERS = listOf('.', '+', '*', '?', '^', '$', '(', ')', '[', ']', '{', '}', '|', '\\')
+private val REGEX_IGNORE_CASE_OPTION = setOfNotNull(RegexOption.IGNORE_CASE)
+private fun String.matchWildcard(value: String, ignoreCase: Boolean = false): Boolean {
+    val pattern = REGEX_SPECIAL_CHARACTERS.filterNot { it == '*' }.fold(
```

Comment:
If you're always filtering "*" from the constant `REGEX_SPECIAL_CHARACTERS`, why not just omit it from the list to begin with?

Reply:
I found the code cleaner this way, but will change it according to your preference.